nearcore/src/lib.rs

```diff
@@ -261,10 +261,13 @@ pub fn init_and_migrate_store(home_dir: &Path, near_config: &NearConfig) -> Arc<
     store
 }
 
-pub fn start_with_config(
-    home_dir: &Path,
-    config: NearConfig,
-) -> (Addr<ClientActor>, Addr<ViewClientActor>, Vec<ArbiterHandle>) {
+pub struct StartResult {
```

Comment:
I would consider refactoring it to `NearNode` type with some API scope (we may start with a single method `start_with_config`, but it might be an opportunity to consider splitting the function into smaller components). I believe [`NodeCluster`](https://github.com/near/nearcore/blob/942e02daf7d75a94db51ef1c4ea9ef27aed459a4/integration-tests/tests/nearcore/node_cluster.rs#L74-L93) could be a great source for inspiration.

@miraclx What do you think?

At the very least, I would rename this struct to `NearNode` since `StartResult` looks odd to me from the public API point of view.